src/zip.h

```diff
@@ -336,16 +336,26 @@ extern int zip_extract_stream(const char *stream, size_t size, const char *dir,
                               void *arg);
 
 /**
- * open zip archive stream into memory.
+ * Opens zip archive stream into memory.
  *
  * @param stream zip archive stream.
  * @param size stream size.
  *
  * @return the zip archive handler or NULL on error
  */
 extern struct zip_t *zip_open_stream(const char *stream, size_t size);
-/** @} */
 
+/**
+ * Deletes zip archive entries.
+ *
+ * @param zip zip archive handler.
+ * @param entries array of zip archive entries to be deleted.
+ * @param num the number of entries to be deleted.
+ * @return the number of deleted entries, or negative number (< 0) on error.
+ */
+extern int zip_entries_delete(struct zip_t *zip, char *entries[],
```

Comment:
Maybe we can replace num by len, so it will be consistent with:
`extern int zip_create(const char *zipname, const char *filenames[], size_t len);`

Also I think it can be `const char *`

Reply:
ok，I will fix it.